src/main/resources/META-INF/gradle-plugins/com.palantir.docker-compose-template.properties

```diff
@@ -0,0 +1 @@
+implementation-class=com.palantir.gradle.docker.DockerComposeTemplatePlugin
```

Comment:
I think maybe we should call this `com.palantir.docker-compose` (the `-template` is superfluous?)